packages/docs/components/select.md

```diff
@@ -340,5 +340,254 @@ Options may be grouped within the Select list to help guide users.
+      <div class="ods-fieldset-flex">
+        <select class="ods-select" data-js-choices id="example-1" name="example-1" required>
+          <option></option>
+          <option value="value-1">Option 1</option>
+          <option value="value-2">Option 2</option>
+          <option value="value-3">Option 3</option>
+          <option value="value-4">Option 4</option>
+          <option value="value-5">Option 5</option>
+          <option value="value-6">Option 6</option>
+        </select>
+        <label class="ods-label" for="example-1">Field label</label>
+      </div>
+    </fieldset>
+  </div>
+
+  ```html
+  <fieldset class="ods-fieldset">
+    <div class="ods-fieldset-flex">
+      <select class="ods-select" data-js-choices id="example-1" name="example-1" required>
+        <option></option>
+        <option value="value-1">Option 1</option>
+        <option value="value-2">Option 2</option>
+        <option value="value-3">Option 3</option>
+        <option value="value-4">Option 4</option>
+        <option value="value-5">Option 5</option>
+        <option value="value-6">Option 6</option>
+      </select>
+      <label class="ods-label" for="example-1">Field label</label>
+    </div>
+  </fieldset>
+  ```
+</figure>
+
+## Multi-Select example
+
+<figure class="docs-example">
+  <div class="docs-example--rendered">
+    <fieldset class="ods-fieldset">
+      <div class="ods-fieldset-flex">
+        <select class="ods-select" data-js-choices id="example-2" name="example-2" multiple required>
+          <option></option>
+          <option value="value-1">Option 1</option>
+          <option value="value-2">Option 2</option>
+          <option value="value-3">Option 3</option>
+          <option value="value-4">Option 4</option>
+          <option value="value-5">Option 5</option>
+          <option value="value-6">Option 6</option>
+        </select>
+        <label class="ods-label" for="example-2">Field label</label>
+      </div>
+    </fieldset>
+  </div>
+
+  ```html
+  <fieldset class="ods-fieldset">
+    <div class="ods-fieldset-flex">
+      <select class="ods-select" data-js-choices id="example-2" name="example-2" multiple required>
+        <option></option>
+        <option value="value-1">Option 1</option>
+        <option value="value-2">Option 2</option>
+        <option value="value-3">Option 3</option>
+        <option value="value-4">Option 4</option>
+        <option value="value-5">Option 5</option>
+        <option value="value-6">Option 6</option>
+      </select>
+      <label class="ods-label" for="example-2">Field label</label>
+    </div>
+  </fieldset>
+  ```
+</figure>
+
+## Option groups example
+
+
+<figure class="docs-example">
+  <div class="docs-example--rendered">
+    <fieldset class="ods-fieldset">
+      <div class="ods-fieldset-flex">
+        <select class="ods-select" data-js-choices id="example-3" name="example-3" required>
+          <option></option>
+          <optgroup label="Group 1">
```

Comment:
Would you mind updating the Overview page to include the note about _all_ options being grouped when using option groups?

I don't think it's a reasonable design guideline, but we're stuck with it until we can ditch choices.

Reply:
> Options may be grouped within the Select list to help guide users. When doing this, consider that Choices will ignore any ungrouped items.

Reply:
@edburyenegren-okta fixed